docs/stylesheets/extra.css

```
@@ -50,6 +56,33 @@
   --md-admonition-bg-color: var(--md-default-bg-color);
 }
 
+/* Header color */
+.md-header {
+    background-color: var(--md-default-bg-color);
+    color: var(--md-typeset-color);
+}
+.md-tabs {
+    background-color: var(--md-default-bg-color);
+    color: var(--md-typeset-color);
+    border-bottom: 1px solid var(--md-border-color);
+}
+.md-search__form:hover {
+    background-color: #0000001f;
+}
+
+/* Site title and logo styling */
+.md-header__button:hover {
+    opacity: 1.0;
+}
+.md-header__topic:first-child {
+    font-weight:400;
+}
```

Use the CSS universal selector (match all elements that are descendants of `.md-header__topic`)
```suggestion
.md-header__topic * {
    font-weight: 400;
}
```